cmd.md

```diff
@@ -0,0 +1,94 @@
+# Runtime Command Line Interface
+
+This document specifies the command line syntax for runtime operations defined in the [Runtime and Lifecycle](runtime.md) specification.
+It is STRONGLY RECOMMENDED that implementations adhere to the command line definition defined below to ensure interoperability.
+This document will not duplicate the information/semantics specified in the [Runtime and Lifecycle](runtime.md) document, rather it just focuses on syntax of the command line.
+
+## Convention
+
+For the purpose of this document, the following syntactical conventions are used:
+
+* `[ ... ]` denotes an optional field
+* `< ... >` denotes a substitution field. 
+The word(s), or phrase, within the `<>` describes the information to be passed-in.
+
+## General Format and Behavior
+
+The general format of all commands MUST be:
+```
+runtime [global-options] action [action-specific-options] [arguments]
```

Comment:
#513 [places limits on the option and action formats][1] so runtimes can identify unrecognised actions as such.  See wking/oci-command-line-api#8 for more discussion.

[1]: https://github.com/opencontainers/runtime-spec/pull/513/files#diff-0c1e5a5baa5649945697a530e09acbf5R10